ui.apps/test/components/commerce/navigation/navigationTest.js

```diff
@@ -36,30 +36,23 @@ describe('Navigation', () => {
                 <div class="header__primaryActions">
                     <button class="navTrigger__root clickable__root" aria-label="Toggle navigation panel">
                         <span class="icon__root">
-                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
-                                <line x1="3" y1="12" x2="21" y2="12"></line>
-                                <line x1="3" y1="6" x2="21" y2="6"></line>
-                                <line x1="3" y1="18" x2="21" y2="18"></line>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">                                
```

Comment:
>you don't need the full SVG here

What I meant to say is that you don't need the SVG *at all* here.

Reply:
Actually I need some svg at places, because I click on the svg in the test.

Reply:
But the actual event is not handled by the SVG, but by the `button` element, at least this is how it should be.

Reply:
Right, the event handler is registered on the button.

Reply:
@dplaton , do you need any more clarification?